display_ui.rb

```diff
@@ -0,0 +1,94 @@
+require_relative 'app'
+
+class DisplayUI
+  def initialize(app)
+    @app = app
+  end
+
+  def list_all_books
+    puts 'Books:-'
+    @app.list_all_books
+  end
+
+  def list_all_peoples
+    puts 'Peoples:-'
+    @app.list_all_peoples
+  end
+
+  def create_book
+    puts 'Title: '
+    title = gets.chomp
+    puts 'Author: '
+    author = gets.chomp
+    @app.create_book(title, author)
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @app.create_teacher(specialization, age, name)
```

Comment:
- [ ] Everything as regards create_teacher, create_student and co should be in the create_person, since they are persons and it would better separate and simplify things.

Reply:
Fix create_person method as per review.